packages/oas-to-har/src/index.js

```diff
@@ -303,33 +303,43 @@ module.exports = (
           } else {
             har.postData.mimeType = contentType;
 
-            // Find all `{ type: string, format: json }` properties in the schema because we need to manually JSON.parse
-            // them before submit, otherwise they'll be escaped instead of actual objects.
-            // We also only want values that the user has entered, so we drop any undefined cleanBody keys
-            const jsonTypes = Object.keys(requestBody.schema.properties).filter(
-              key => requestBody.schema.properties[key].format === 'json' && cleanBody[key] !== undefined
-            );
+            // Handle arbitraty JSON input via a string.
+            // In OAS you usually find this in an application/json content type.
+            //   with a schema type=string, format=json.
+            // In the UI this is represented by an arbitrary text input
+            // This ensures we remove any newlines or tabs and use a clean json block in the example
+            if (requestBody.schema.type === 'string') {
+              har.postData.text = JSON.stringify(JSON.parse(cleanBody));
```

Comment:
The diff sucks on this. Really all I added was this line, and indented everything else that existed under the else.

The problem was that line 318 in the right hand file was failing because these schemas have no properties, and Object.keys would error on undefined.